src/clj/qbits/spandex.clj

```diff
@@ -202,30 +203,66 @@
                (response-headers response)
                (.getHost response))))
 
+(defn response-ex->response
```

Comment:
naming is still subject to change, I am not happy with what's in master atm, but I am busy with other stuff right now :)